lib/src/Base/Func/UniVariateFunctionImplementation.cxx

```diff
@@ -91,7 +91,10 @@ Graph UniVariateFunctionImplementation::draw(const Scalar xMin,
     data(i, 0) = x;
     data(i, 1) = operator()(x);
   }
-  Curve curve(data, "red", "solid", 2, getName());
+  Curve curve(data);
+  curve.setLineStyle("solid");
```

Comment:
According to https://github.com/openturns/openturns/blob/414b0908346a84a60f86abcdcfe81e74beba4618/lib/src/Base/Common/ResourceMap.cxx#L704, "solid" is the default. This is why I believe that it is not necessary.

Reply:
And is 2 the default for the linewidth ?

Reply:
It is not, so that needs to stay. Thanks for noticing this, I have changed the code accordingly.